pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -1221,15 +1176,6 @@
     "spdx": "MPL-2.0",
     "vendorHash": "sha256-smeySV1kReZyF9bRCunEr89IV219f9845wcHHI1zFz8="
   },
-  "vercel": {
-    "hash": "sha256-/LHyNxal5Il/UzXdCKfVRzK/VVfSYMgoeKerWsedmho=",
-    "homepage": "https://registry.terraform.io/providers/ondrejsika/vercel",
-    "owner": "ondrejsika",
-    "repo": "terraform-provider-vercel",
```

Comment:
This case is even weirder. Even github detect the license here: https://github.com/vercel/terraform-provider-vercel/blob/main/LICENSE

Reply:
https://api.github.com/repos/ondrejsika/terraform-provider-vercel/license